lib/ClimaLandSimulations/src/ClimaLandSimulations.jl

```diff
@@ -1,57 +1,34 @@
 module ClimaLandSimulations
 
-import SciMLBase
-import ClimaTimeSteppers as CTS
-using ClimaCore
-import CLIMAParameters as CP
+using CairoMakie
+using DataFrames
+using LaTeXStrings
+using PlotUtils: optimize_ticks
+using HTTP
 using Statistics
+using JSON
 using Dates
-using Insolation
-using StatsBase
-using Interpolations
+import ClimaTimeSteppers as CTS
 using ClimaLand
 using ClimaLand.Domains: Column
-using ClimaLand.Soil
-using ClimaLand.Soil.Biogeochemistry
-using ClimaLand.Canopy
-using ClimaLand.Canopy.PlantHydraulics
 import ClimaLand.Parameters as LP
-import ClimaComms
-using Unitful: R, L, mol, K, kJ, °C, m, g, cm, hr, mg, s, μmol, Pa, W, mm, kPa
-using UnitfulMoles: molC
-using Unitful, UnitfulMoles
-using HTTP
-using JSON
-using ArtifactWrappers
-using DelimitedFiles
-using Thermodynamics
-using Formatting
-using CairoMakie
-using LaTeXStrings
-using PlotUtils: optimize_ticks
-using DataFrames
-
-@compound CO₂
-const FT = Float64
 
+# Directory paths
+climalandsimulations_dir = pkgdir(ClimaLandSimulations)
 climaland_dir = pkgdir(ClimaLand)
-savedir =
-    joinpath(climaland_dir, "experiments", "integrated", "fluxnet/figures/")
+
+# ClimaParameters
 include(joinpath(climaland_dir, "parameters", "create_parameters.jl"))
+const FT = Float64
 earth_param_set = create_lsm_parameters(FT)
```

Comment:
FT shouldn't be defined here. If we move `earth_param_set` inside the driver `fluxnet_simulation` we won't need it here

Reply:
done